### PR TITLE
Refactored HTTP dependency parsing to use dedicated parsers

### PR DESCRIPTION
**Pull request by TomMilos**

### CODE REVIEW

File: Src/DependencyCollector/Shared/HttpParsers/AzureBlobHttpParser.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
+﻿namespace Microsoft.ApplicationInsights.DependencyCollector.HttpParsers
```
Comment (SergeyKanzhelev) [inline Src/DependencyCollector/Shared/HttpParsers/AzureBlobHttpParser.cs]: We typically place internal classes under `Implementation` folder

Comment (TomMilos) [inline Src/DependencyCollector/Shared/HttpParsers/AzureBlobHttpParser.cs]: Done

**Comment by SergeyKanzhelev** [comment]
@TomMilos you'd need to rebase locally so only your changes will be included

**Comment by TomMilos** [comment]
Branch got corrupted after merges and rebasing

**Pull request closed without merging**